### DISCUSSION TITLE
Hope a simple full example

### DISCUSSION THREAD

**Discussion by NocoleNe** [enable3d/enable3d#312] [Help / Questions] [answered]
Great work, I am looking for the 3D solution of phaser3, and find your project, great.

I'm developing with haxe language and phaser, https://github.com/sandersn/downlevel-dts.
where can find simple like hellow word simple example to start?

In fact, I would like to see a small independent example containing the configuration package.json  file, I can run it myself.

my lib like sthis 
`  "dependencies": {
    "@types/matter-js": "0.17.6",
    "@types/three": "0.141.0",
    "matter-js": "0.17.1",
    "phaser": "^3.55.2",
    "poly-decomp": "^0.3.0",
    "three": "0.141.0",
    "three-csg-ts": "^3.1.10"
  }`

But where  to find the "enable3d"

**Comment by vitalspace** [comment, marked as answer]
maybe

https://github.com/enable3d/enable3d-phaser-project-template

or all examples 

https://enable3d.io/examples.html
